integration/airflow/openlineage/airflow/macros.py

```diff
@@ -31,7 +31,9 @@ def lineage_run_id(task: "BaseOperator", task_instance: "TaskInstance"):
     )
 
 
-def lineage_parent_id(run_id: str, task: "BaseOperator", task_instance: "TaskInstance"):
+def lineage_parent_id(
+    task: "BaseOperator", task_instance: "TaskInstance", run_id: typing.Optional[str] = None
+):
```

Comment:
I think the run_id will always be deterministic and there will be no need to overwrite it, but maybe I am missing something. Let's wait a while so the others have a chance to look at it

Reply:
Using macro is not mandatory, therefore I think there's no need to allow users to pass `run_id` as optional argument in case their logic is different.